sentry-spring/build.gradle.kts

```diff
@@ -35,6 +35,8 @@ tasks.withType<KotlinCompile>().configureEach {
 dependencies {
     api(project(":sentry"))
     compileOnly(Config.Libs.springWeb)
+    compileOnly(Config.Libs.springWebflux)
+    compileOnly(Config.Libs.springBootStarterWebflux)
```

Comment:
I'd probably extract this integration into its own module.

https://github.com/getsentry/sentry-java/issues/640#issuecomment-711157472

wdyt @bruno-garcia @maciejwalkowiak

Reply:
given that's `compileOnly`, seems OK. It's a no-op if the dep isnt' there at runtime so it doesn't drag it in.

The code has to work without having this at runtime though

Reply:
It may depend on Spring Webflux but it may not depend on any Spring Boot component. I think ideally Webflux integration stays in `sentry-spring` but then we need to make `@EnableSentry` annotation smart enough to figure out if Webflux is on the classpath or not.

Reply:
I'll then resolve the conversation with that conclusion, thank you for all your comments